apiserver/plane/api/views/issue.py

```
@@ -141,6 +141,8 @@ class IssueAPIEndpoint(BaseAPIView):
     def get_queryset(self):
         filters = issue_filters(self.request.query_params, "GET")
         base_filter = filters.pop('base', None)
+        custom_properties = filters.pop('custom_properties', None)
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)